packages/sdk/src/4.0/models.ts

```diff
@@ -3247,6 +3415,17 @@ export interface IDialect {
   has_ssl_support?: boolean;
 }
 
+export interface IDialectDriverVersion {
```

Comment:
FWIW in the next release this will be called `IDialectDriverNameVersion` but hopefully no one will start consuming it before it's renamed in 25.8